PrivateCollections/Germany/Neukirchen003.xml

```diff
@@ -0,0 +1,1145 @@
+                        <textLang mainLang="gez"></textLang>
+                    </msItem>
+                    <msItem xml:id="ms_i1.3">
+                      <locus from="118r" to="124v"></locus>
+                      <title type="complete" ref="LIT2362Songof"></title>
+                      <textLang mainLang="gez"></textLang>
+                    </msItem>
+                    <msItem xml:id="ms_i1.4">
+                      <locus from="125r" to="137v"></locus>
+                      <title type="complete" ref="LIT2509Weddas"></title>
+                      <textLang mainLang="gez"></textLang>
+                    </msItem>
+                  </msItem>
+
+               </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf">138</measure>
+                        <measure unit="quire">13</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>193</height>
+                              <width>118</width>
+                           </dimensions>
+                           <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                             <height>193</height>
+                             <width>116</width>
+                           </dimensions>
+                           <note corresp="#leafdim">
+                             Data on leaves dimensions
+                             taken from <locus target="#27r"></locus>.
+                           </note>
+                        </extent>
+                        <foliation>
+                          <list>
+                            <item xml:id="fol1" resp="DE">
+                              <locus from="1r" to="138v"></locus>
+                              Foliation in pencil in the right bottom corner of the folio. This is the foliation used as reference in this record.
+                            </item>
+                          </list>
+                        </foliation>
+                        <condition key="good">Stains of water and dirt throughout the manuscript. First two quires have been
+                        reattached to the binding with a green thread. The textblock has been reattached to the binding. The leather
+                      cover of the binding is partially cracked, rubbed off and damaged by insects.</condition>
+                         </supportDesc>
+                     <layoutDesc>
+                           <layout corresp="#ms_i1.1 #ms_i1.2 #ms_i1.3" columns="1" writtenLines="24">
+                             <locus from="3r" to="124v"></locus>
+                             <ab type="ruling">Ruling and pricking are visible.</ab>
```

Comment:
It would of course be nice (and in this case, possible) to have the ruling pattern (I also always procrastinate this...).

Reply:
done